packages/editor/src/package/editor.tsx

```diff
@@ -49,7 +49,7 @@ const emptyState = {
 /** For exporting the editor */
 export function SerloEditor(props: SerloEditorProps) {
   const { children, pluginsConfig, initialState, language = 'de' } = props
-  const { basicPluginsConfig, customPlugins = [] } = pluginsConfig
+  const { basicPluginsConfig, customPlugins = [] } = pluginsConfig || {}
```

Comment:
Would be great if `basicPluginsConfig` could get its default value here as well. See other comment for my thinking.